### ISSUE TITLE
cTemplateId="columnFilter_columnName" not work

### ISSUE THREAD

**Issue by abdalahshaban07** [coreui/coreui-free-angular-admin-template#249]
```
<ng-template cTemplateId="columnFilter_category">
    <select
      aria-label="Multiple select example"
      cSelect
      multiple
      size="3"
      sizing="lg"
    >
      <option>Select multiple options</option>
      <option value="1">One</option>
      <option value="2">Two</option>
      <option value="3">Three</option>
      <option value="4">Four</option>
    </select>
  </ng-template>
```

// column
```
{
      key: 'category',
      label: 'Category',
      sorter: true,
      filter: true,
      _style: { width: '20%' }
    },
```

/// table 
  [columnFilter]="true"

**Comment by xidedix** [comment]
@abdalahshaban07 
`cTemplateId` directive needs `SharedModule`, make sure you've got it imported

**Comment by abdalahshaban07** [comment]
sharedModule imported and use cTemplateId="tableData"  work but cTemplateId="columnFilter_category" not work

**Comment by abdalahshaban07** [comment]
solved by update "@coreui/angular-pro": "~4.2.0", to "@coreui/angular-pro": "~4.2.38",
